frontend/src/layouts/ProtectedLayout.vue

```diff
@@ -6,16 +6,30 @@ import SideNav, {
     type ISideNavItem,
 } from '@/components/common/SideNav.vue';
 import sideNavData from '@/static/sideNav.json';
-import { isApplicationSelected } from '@/store/ApplicationState';
+import { FAM_APPLICATION_NAME } from '@/store/Constants';
+import { isApplicationSelected,selectedApplicationShortDisplayText } from '@/store/ApplicationState';
 
 const navigationData = ref<[ISideNavData]>(sideNavData as any);
 
+// Show and hide the correct sideNav btn based on the application
+const sideNavAddPermissionOption = () => {
+    if(selectedApplicationShortDisplayText.value === FAM_APPLICATION_NAME) {
+        disableSideNavOption('Add user permission', true);
+        disableSideNavOption('Add application admin', false);
+    } else {
+        disableSideNavOption('Add application admin', true);
+        disableSideNavOption('Add user permission', false);
+    }
+};
+
 onMounted(() => {
-    disableSideNavOption('Add user permission', !isApplicationSelected.value);
+    if(isApplicationSelected.value) {
+        sideNavAddPermissionOption()
```

Comment:
I wonder maybe we can call it setSideNavAddPermissionOption to be more like a method name? thanks!!

Reply:
ok, I'll change the name